### ISSUE TITLE
2.x: zcml condition for including archetypes related code is not specific enough

### ISSUE THREAD

**Issue by ale-rt** [plone/plone.app.caching#72]
There are some `zcml:condition="installed Products.Archetypes"` that might conflict with BBB patches providing aliases for the Python3 migration.
E.g. if you have some code like:
```
alias_module("Products.Archetypes", missing_archetypes)
```

That will make the startup of the instance fail with:
```
    File ".../plone.app.caching-2.0.6-py2.7.egg/plone/app/caching/caching.zcml", line 114.8-114.92
    ValidationError: ImportError: Couldn't import Products.Archetypes.Field, No module named Field
```
I would propose to change the zcml conditional with 
`zcml:condition="installed Products.Archetypes.atapi"` which has a very low chance of being present in the aliased module.

**Comment by ale-rt** [comment]
See also: https://github.com/plone/plone.restapi/issues/1004

**Comment by mauritsvanrees** [comment]
Fixed by yourself in #73.
